spec/controllers/ctc/questions/stimulus_two_controller_spec.rb

```diff
@@ -9,7 +9,7 @@
   end
 
   describe "#update" do
-    it "saves 0 as the amount for stimulus 2 and redirects to the stimulus-received" do
+    it "saves 0 as the amount for stimulus 2 and redirects to stimulus owed" do
```

Comment:
Same as above -- by saying they did not receive stimulus and they have money they are eligible for, they should be sent to stimulus-owed to claim their owed stimulus should they so choose.